src/dbus-objects/wireless.c

```diff
@@ -391,6 +391,23 @@ ni_objectmodel_get_wireless_request_net(const char *ifname, ni_wireless_network_
 	if (ni_dbus_dict_get_uint32(var, "channel", &value))
 		net->channel = value;
 
+	if (ni_dbus_dict_get_string(var, "frequency-list", &string)) {
+		ni_string_array_t errors = NI_STRING_ARRAY_INIT;
+		ni_stringbuf_t tmp = NI_STRINGBUF_INIT_DYNAMIC;
+
+		if (!ni_wireless_frequency_list_parse_string(string, &net->frequency_list, &errors)) {
+			if (!dbus_error_is_set(error))
+				dbus_set_error(error, DBUS_ERROR_INVALID_ARGS,
+						"%s: invalid frequency-list value %s",
+						ni_stringbuf_join(&tmp, &errors, ","),
+						ifname);
```

Comment:
-> ifname context first, then (space separated) erroneous frequencies:
```suggestion
				dbus_set_error(error, DBUS_ERROR_INVALID_ARGS,
						"%s: invalid frequency-list value %s",
						ifname, ni_stringbuf_join(&tmp, &errors, " "));
```

Reply:
good catch, thx!